modules/main-rules/pages/transport.adoc

```diff
@@ -5,35 +5,44 @@ We explain special abilities in more detail at xref:special-rules:transport-x.ad
 
 Units that have the {transport} special ability can carry other units across the battlefield.
 
-* To embark or disembark other units (or cargo) it costs the {transport} unit 5cm of its movement.
+* To embark or disembark other units (or cargo) costs the {transport} unit 5cm of its movement.
 * An embarking or disembarking unit must also pay 5cm of its movement.
 * You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
-* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example. 
+* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
 Obviously, once you disembark them then the normal rules apply again.
 
+[NOTE]
+.Placement of disembarked units
+====
+When you disembark units, measure their movement from the base of the {transport} unit.
+If you do not have enough movement to clear the {transport} unit's base then place the disembarked unit in base contact with the {transport} unit.
+====
```

Comment:
I'm editing this topic at the moment and will share my efforts soon. But on these particular lines, I wonder as follows:

As written, might this permit cases where players could have sacrificed some movement, but don't because the rules say they can put the units in base contact anyway - so they use that to get a 'free' gain?

In general, I feel like perhaps there should be some concept of not exceeding the 'calculated maximum move you could get from the transport plus the embarked unit in combination' -- even if this means you must move transport unit a bit less or something to compensate rather than adding on the width of the disembarked unit's base and thus going beyond the calculated maximum move.

If there is something in this, I'd be happy for us to create an issue to tackle this properly later - just wanted to raise the point and get your thoughts.

Reply:
> As written, might this permit cases where players could have sacrificed some movement, but don't because the rules say they can put the units in base contact anyway - so they use that to get a 'free' gain?

As far as I can see, this would require them to have a base longer than 5 cm. 

Don't forget that the unit is **already** sacrificing 5 cm of move just to be able to disembark. This is another reason why I automatically assumed that you 'deploy then measure' - as far as I was concerned, the unit had paid a movement penalty to get out, why would the unit have to pay a **second** movement penalty by measuring from the transport to 'move' out?

As a reminder, don't forget that what we currently have is already going to be regarded as a controversial rules change by some players, and if they've played 'deploy then measure' for a long time it's going to be unpopular, at best.

> In general, I feel like perhaps there should be some concept of not exceeding the 'calculated maximum move you could get from the transport plus the embarked unit in combination' -- even if this means you must move transport unit a bit less or something to compensate rather than adding on the width of the disembarked unit's base and thus going beyond the calculated maximum move.

My gut reaction is that this part of the map is marked 'Here be Dragons'. It's likely to be hard to write clear concise rules for, the overall subject is already controversial/generates lots of arguments, it's unlikely to get widespread support from the players (and that includes me), and it adds even more inconsistencies with Hitch. Or you apply it to Hitch as well, and screw over the Orks even more.

Paraphrasing...

'You know that unstoppable Battlefortress war engine that isn't slowed down at all when Da Boyz leap suicidally off the side?'
'Yer.'
'Well, it just slowed down by 4 cm because some of Da Boyz want to leap suicidally off the front instead of the side.'

P.S. In places where I'd normally use italics for emphasis I'm using bold on Github for consistency with the style guide.

Reply:
>Don't forget that the unit is already sacrificing 5 cm of move just to be able to disembark.

I always rationalised the 5cm cost as a cost in **time** to embark/disembark, which seems reasonable and logical to me - not a 'compensation' for the width of the base, as you appear to see it. Though I can see that perhaps that makes sense from the point of view of game mechanics, but then why pay it twice?

But yeah, okay, let's skip this. The very fact that this part of the rules is controversial and creates arguments is exactly why I'm thinking that perhaps we could at least suggest 'ways to play this to make it more consistent or fair' -- but that could be in another section in future perhaps, especially as it might be hard to make concise, as you say.

It really irritates me when people 'conveniently' gain extra move for their units by virtue of the width of their base - so personally I tend to play it so that no individual unit can move farther in total than the maximum you can calculate that they should move. That seems the only fair way to me, and if that means moving your Battlefortress back a bit then so be it. If the Orks are massively disadvantaged in some way, then fix that rather than focus on loss of a gain that wasn't earned anyway.

But yes, different systems, different views. I've taken a view that seems the most consistent and fair to me.

I'll resolve this as 'nothing to do here', but if we do ever do a section for 'suggestions you can pick and choose for things that might otherwise be contentious or that don't render down to nice concise rules' then I think this is a contender for more attention!